### PR TITLE
CC specified users (i.e. staff) on all Activity emails

### LINKED ISSUE #3557: Copy admins at amo on all activity emails
Splitting out #3349 to make it easier to track

Add a new permission and group for users that get a copy of all activity emails (with reply tokens)

### PR DESCRIPTION
**Pull request by eviljeff**
Fixes #3557

### CODE REVIEW

File: src/olympia/activity/utils.py [hunk 1]
```diff
@@ -136,8 +138,21 @@ def log_and_notify(action, comments, note_creator, version):
         alog.user for alog in ActivityLog.objects.for_version(version)]
     # Collect add-on authors (excl. the person who sent the email.)
     addon_authors = set(version.addon.authors.all()) - {note_creator}
-    # Collect reviewers on the thread (again, excl. the email sender)
-    reviewer_recipients = set(log_users) - addon_authors - {note_creator}
+    # Collect staff that want a copy of the email
+    try:
+        cc_group = models.Group.objects.get(name='Activity Mail CC')
+        staff_cc = [user for user in cc_group.users.all()]
```
Comment (EnTeQuAk) [inline src/olympia/activity/utils.py]: You're not using `cc_group` later, maybe something like `UserProfile.objects.filter(groups__name='Activity Mail CC')` is more performant than fetching the group first and the users later (two queries, two round trips). Though "performant" is relative here, it's a string match on `name` but we don't have too many groups imho so that should be fine.


Also, we should start (or do we already do that somewhere?) using constants for group names like these and put them into `olympia.constants.{users,group}` or something like that.

Comment (eviljeff) [inline src/olympia/activity/utils.py]: Hmm.  I had it as a single line of `staff_cc = [user for user in models.Group.objects.get(name='Activity Mail CC').users.all()]` but line wrapping made it hard to follow btw.

Regarding making it more efficient - do the relative counts make a difference to filtering via User->GroupUser->Group rather than queries of Group then GroupUser->User?  User is ~5.2 million; GroupUser likely a few hundred; Group 70.

As I've been banging on about using constants instead of names we probably should, heh.  Though I'd note it's only in exceptional circumstances we should be looking up Groups, or checking for group membership, because in 99% of cases it's the permission a known user(s) has we are interested in.  In this case searching through said 5.2 million users to find ones that indirectly have a permission because they are members of a group that has matching rules, including *, is even more crazy though.

Comment (diox) [inline src/olympia/activity/utils.py]: Ultimately the query being made will be similar - MySQL should be smart enough to look at which groups match first, I think. I think I prefer the version with the `cc_group` variable.

Comment (eviljeff) [inline src/olympia/activity/utils.py]: looks like this now (not pushed yet):
```python
    # Collect reviewers/others involved with this version.
    log_users = {
        alog.user for alog in ActivityLog.objects.for_version(version)}
    # Collect add-on authors (excl. the person who sent the email.)
    addon_authors = set(version.addon.authors.all()) - {note_creator}
    # Collect staff that want a copy of the email
    staff_cc = set(UserProfile.objects.filter(groups__name=ACTIVITY_MAIL))
    # If task_user doesn't exist that's no big issue (i.e. in tests)
    try:
        task_user = {get_task_user()}
    except UserProfile.DoesNotExist:
        task_user = set()
    # Collect reviewers on the thread (excl. the email sender and task user for
    # automated messages).
    reviewers = ((log_users | staff_cc) - addon_authors - task_user -
                 {note_creator})
```

**Comment by eviljeff** [comment]
changes made on @EnTeQuAk suggestion
